int/api/websites/all.go

```diff
@@ -80,24 +74,25 @@ func Registry(client *node.Client, candidateDatastoreKeys [][]byte) ([]*models.R
 		return nil, fmt.Errorf("metadata reaching on dnsContractStorers failed : %w", err)
 	}
 
-	dates := make([]dateOnChain, len(metadatas))
-
-	for index, metadata := range metadatas {
-		var date dateOnChain
-
-		_ = json.Unmarshal(metadata.CandidateValue, &date)
-		dates[index] = date
-	}
+	registryResult := make([]*models.Registry, len(metadatas))
 
-	registryResult := make([]*models.Registry, len(dates))
-	for index, date := range dates {
+	for index := 0; index < len(metadatas); index++ {
 		registryResult[index] = &models.Registry{
-			Name:      strings.Split(recordKeys[index], recordKey)[1],
-			Address:   metadataKeys[index].Address,
-			CreatedAt: time.Unix(date.CreateDate/secondsToMilliCoeff, 0).Format(dateFormat),
-			UpdatedAt: time.Unix(date.UpdateDate/secondsToMilliCoeff, 0).Format(dateFormat),
+			Name:     strings.Split(recordKeys[index], recordKey)[1],
+			Address:  metadataKeys[index].Address,
+			Metadata: ByteArrToString(metadatas[index].CandidateValue),
```

Comment:
If you want to convert it to string, why not using string(metadatas[index].CandidateValue[:])

Reply:
I chosed to use a custom function and not string because string is the equivalent of fromCharCode and will return a string created from the specified sequence of UTF-16 code units. (expected output: "½+¾=") I wanted to have an output like this : 108,109,13,194,142,194,132,1,0,0,195 to be easier to handle on front end side and avoid to be reprocessed with charCodeAt()

Reply:
changed